atm.js

```diff
@@ -1,21 +1,75 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$("input").eq(0).attr("id", "cInBox"); //create id for checking account input box
+$("input").eq(1).attr("id", "cDeBut"); //create id for checking deposit button
+$("input").eq(2).attr("id", "cWiBut"); //create id for checking withdraw button
+$("input").eq(3).attr("id", "sInBox"); //create id for savings account input box
+$("input").eq(4).attr("id", "sDeBut"); //create id for checking deposit button
+$("input").eq(5).attr("id", "sWiBut"); //create id for checking withdraw button
+$(".balance").eq(0).attr("id", "cBalTot");//create id for checking balance total
+$(".balance").eq(1).attr("id", "sBalTot");//create id for savings balance total
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+// Removes the $ from the balance
+cBalTot.innerHTML.replace("$", "")
+// sum checking balance
+  var cBal = 0;
+  cBalTot.innerHTML.replace("$", "");
+$("#cDeBut").on("click", function(){
+  var depCBal = parseFloat($("#cInBox").val());
+  cBal = (depCBal + cBal);
+  $("#cBalTot").text("$"+cBal);
+})
+// withdraw from current balance
+$("#cWiBut").on("click", function(){
+  var wdCBal = parseFloat($("#cInBox").val());
+  cBal = (cBal - wdCBal);
+  $("#cBalTot").text("$"+cBal);
+})
+// sum savings balance
+  var sBal = 0;
+  sBalTot.innerHTML.replace("$", "");
+$("#sDeBut").on("click", function(){
+  var depCBal = parseFloat($("#sInBox").val());
+  sBal = (depCBal + sBal);
+  $("#sBalTot").text("$"+sBal);
+})
+// withdraw from savings balance
+$("#sWiBut").on("click", function(){
+      var wdSBal = parseFloat($("#sInBox").val());
+      sBal = (sBal - wdSBal);
+      $("#sBalTot").text("$"+ sBal);
 
-}
 
-function checkingWithdrawal(amount){
+      // if withdraw is more than balance, cannot withdraw from savings
+/*    if ($("sInBox").val() <= sBal){
```

Comment:
I think using variables to hold the amounts would make this code easier to read/write, e.g:

```js
var checkingBalance = parseFloat($("#cInBox").val());
var savingsBalance   = parseFloat($("#sInBox").val());
var withdrawlAmount = parseFloat($("#sInBox").val());
var totalBalance = checkingBalance + savingsBalance;

if (withdrawlAmount > totalBalance) {
  // can't overdraft
} else {
  // can overdraft
}
```